walkthrough.ipynb

```diff
@@ -345,6 +329,134 @@
     "summary_df = df.agg([\"min\", \"max\", \"mean\", \"median\"])\n",
     "print(summary_df)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cross-Schedule Predictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "Using the same building blocks, we can also train a model to predict cross-schedule PK response.\n",
+    "\n",
+    "In this case, we only run one outer fold, but still train 5 models within this fold. Each of the \n",
+    "models has the same test split (train and validate on 3-week dosing schedule, test on 1-week dosing \n",
+    "schedule), but different train/validation splits (split randomly).\n",
+    "\n",
+    "In this code, we have implemented the training and prediction of the split described above. The\n",
+    "original code creates 2 additional test sets:\n",
+    "(1) test_ineterp - the same test set, but with interpolated time points at 6 hour intervals. This was\n",
+    "    done to be able to generate predictions at a higher resolution.\n",
+    "(2) test_nodosing - the same test set with all dosing information masked (replaced with 0). This was\n",
+    "    used to produce the analysis described in Figure 4B.\n",
+    "\"\"\"\n",
+    "eval_results_all = {}\n",
+    "# we retained the outer fold loop for similarity of code. This can be replaced with \n",
+    "# fold = CROSS_SCHED_FOLD, as it is only used for setting the seed and naming files\n",
+    "for fold in CROSS_SCHED_FOLD:\n",
```

Comment:
I think a for loop over one element is confusing. Just get rid of it and make `CROSS_SCHED_FOLD = 999` instead of `CROSS_SCHED_FOLD = [999]`